aepsych/benchmark/test_functions.py

```diff
@@ -272,120 +279,121 @@ def new_novel_det(x, scale_factor=1.0):
     return (x[..., 1] - locs) / scale
 
 
-def cdf_new_novel_det(x, scale_factor=1.0):
+def cdf_new_novel_det(x: torch.Tensor, scale_factor: float = 1.0) -> torch.Tensor:
     """Get the cdf for 2D synthetic novel_det(frequency) function
         Keyword arguments:
-    x -- array of shape (n,2) of locations to sample;
+    x -- tensor of shape (n,2) of locations to sample;
          x[...,0] is frequency from -1 to 1; x[...,1] is intensity from -1 to 1
     scale factor -- scale for the novel_det function, where higher is steeper/lower SD
     """
-    return norm.cdf(new_novel_det(x, scale_factor))
+    z = new_novel_det(x, scale_factor)
+    normal_dist = torch.distributions.Normal(0, 1)  # Standard normal distribution
+    return normal_dist.cdf(z)
 
 
-def new_novel_det_channels_params(channel, scale_factor=1.0, wave_freq=1, target=0.75):
+def new_novel_det_channels_params(channel: torch.Tensor, scale_factor: float = 1.0, wave_freq: float = 1, target: float = 0.75) -> Tuple[torch.Tensor, torch.Tensor]:
```

Comment:
This line is probably too long and should be formatted. Don't worry about this one but in the future we are going to do our best to ensure a consistent formatting across the codebase using black, flake8, and uimport.